File: python/meteroid/models/revocation_request.py
# this file is @generated
import dataclasses
import typing as t

from ..serialization import BaseModel


@dataclasses.dataclass
class RevocationRequest(BaseModel):
    """Token revocation request"""

    token: str
    """The token to revoke"""

    token_type_hint: t.Optional[str] = None
    """Optional hint about the token type (access_token or refresh_token)"""

File: go/revocation_request.go
// this file is @generated
package meteroid

// Token revocation request
type RevocationRequest struct {
	// The token to revoke
	Token string `json:"token"`

	// Optional hint about the token type (access_token or refresh_token)
	TokenTypeHint *string `json:"token_type_hint,omitempty"`
}
